working/0546-patterns/patterns-feature-specification.md

```diff
@@ -579,6 +594,28 @@ It is a compile-time error if:
     don't have primitive equality, it is possible to have redundant keys and the
     compiler won't detect it.*
 
+*   There is more than one `...` element in the map pattern. *It can
+    appear anywhere in the map, but there can only be zero or one.*
```

Comment:
The italicized text here contradicts the bullet below.  Is the `...` allowed to appear anywhere in the map, or not?

Reply:
Oops, copy-pasted from list and didn't fix. :) Fixed.